pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -76,7 +76,8 @@ blocked = true`)
 	assert.Equal(t, 2, len(registries))
 
 	var reg *Registry
-	reg = FindRegistry("registry.com/image:tag", registries)
+	reg, err = FindRegistry(nil, "registry.com/image:tag")
+	assert.Nil(t, err)
 	assert.NotNil(t, reg)
```

Comment:
(Some of these should be `require.` so that future asserts don’t fail with `nil` references. But that’s trivial to retrofit if the tests ever fail, so not blocking.

A fair amount of this would probably be smaller if the tests were table-driven, but again, this works just fine, absolutely non-blocking.)